jake/types/results_decoder.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Sonatype Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+
+from jake.types.coordinateresults import CoordinateResults
+
+class ResultsDecoder(json.JSONDecoder):
+  def __init__(self):
+    json.JSONDecoder.__init__(self, object_hook=self.dict_to_object)
+
+  def dict_to_object(self, dictionary):
```

Comment:
I will try to remember this very elegant solution to json serialization

Reply:
@gmohre is who showed me this, and also who endured my whole "Wow this is so much easier in Golang" associated rant